### DISCUSSION TITLE
Does SO support offline mode?

### DISCUSSION THREAD

**Discussion by ryguy-vt** [Security-Onion-Solutions/securityonion#1740] [Unsupported Versions] [answered]
Does security onion (sensor node) still connect logs after the internet connection is lost. Example: I set up a sensor/forward node on a local LAN, however the connection between my sensor and the master node is lost. Will the sensor node continue to collect data on the agents until the connection is reestablished? Then send all the logs once the internet is up and running?

**Comment by weslambert** [comment, marked as answer]
Yes, the sensor will still collect data, which will be forward to the
manager once the connection is back up.

Thanks,
Wes

On Thu, Oct 29, 2020 at 4:06 PM ryguy-vt <notifications@github.com> wrote:

> Does security onion (sensor node) still connect logs after the internet
> connection is lost. Example: I set up a sensor/forward node on a local LAN,
> however the connection between my sensor and the master node is lost. Will
> the sensor node continue to collect data on the agents until the connection
> is reestablished? Then send all the logs once the internet is up and
> running?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/1740>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEAM3KCUR2BAMXHYHTFSUXLSNHDMHANCNFSM4TEGGKOQ>
> .
>


-- 
https://twitter.com/therealwlambert
https://securityonion.net/
